pygeoapi/processes/location_info_rasterstats.py

```diff
@@ -0,0 +1,122 @@
+# =================================================================
+#
+# Authors: Tom Kralidis <tomkralidis@gmail.com>
+#
+# Copyright (c) 2019 Tom Kralidis
+#
+# Permission is hereby granted, free of charge, to any person
+# obtaining a copy of this software and associated documentation
+# files (the "Software"), to deal in the Software without
+# restriction, including without limitation the rights to use,
+# copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following
+# conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# =================================================================
+
+import logging
+
+
+from pygeoapi.process.base import BaseProcessor, ProcessorExecuteError
+from .algorithms.location_info import get_location_info
+import json
+
+LOGGER = logging.getLogger(__name__)
+
+#: Process metadata and description
+PROCESS_METADATA = {
+    'version': '0.1.0',
+    'id': 'location-info-rasterstats',
+    'title': {
+        'en': 'Location info of a COG using rasterstats',
+        'de': 'Standortinformation eines COGs mit rasterstats'
+    },
+    'description': {
+        'en': 'Get information of a location of an publicly accessible COG. Only queries the data from the first raster band.',
+        'de': 'Gibt Informationen über einen Standort eines öffentlich zugänglichen COGs zurück. Fragt nur Daten vom ersten Rasterband ab.'
```

Comment:
`informationen über einen standort` sounds a bit to general to me. we request the raster values of a cog so maybe
```suggestion
        'de': 'Fragt Rasterwerte eines öffentlich zugänglichen COG basierend auf Input-Koordinaten ab. Dabei wird nur das erste Band abgefragt.'
```